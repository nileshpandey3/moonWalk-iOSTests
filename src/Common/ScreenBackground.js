import React from "react";
import LinearGradient from "react-native-linear-gradient";

export default ({ children, style }) => (
  <LinearGradient
    colors={["#373468", "#222437"]}
    start={{ x: 0.0, y: 0.25 }}
    end={{ x: 0.5, y: 1.0 }}
    style={style}
  >
    {children}
  </LinearGradient>
);
