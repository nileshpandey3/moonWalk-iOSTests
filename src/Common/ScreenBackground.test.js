import React from "react";
import ScreenBackground from "./ScreenBackground";
import renderer from "react-test-renderer";

describe("screenBackground", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<ScreenBackground />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
