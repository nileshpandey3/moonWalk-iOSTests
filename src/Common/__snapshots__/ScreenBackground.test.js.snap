// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`screenBackground renders correctly 1`] = `
<BVLinearGradient
  colors={
    Array [
      4281807976,
      4280427575,
    ]
  }
  endPoint={
    Object {
      "x": 0.5,
      "y": 1,
    }
  }
  locations={null}
  startPoint={
    Object {
      "x": 0,
      "y": 0.25,
    }
  }
/>
`;
